include/boost/sml/back/state_machine.hpp

```diff
@@ -406,6 +406,11 @@ class sm {
   sm(sm &&other) : deps_{other.deps_}, sub_sms_{other.deps_} {}
 
   sm(const sm &) = delete;
+  sm &operator=(sm &&other) {
```

Comment:
it's in the final sml.hpp but not here causing pph.h to produce incorrect sml.hpp